File: Python Exams/Basic Exam Collection/Basics Online Exam - 6 and 7 July 2019/4. Renovation/4. Renovation.py
from math import ceil

height, width, percent = [int(input()) for _ in range(3)]
wall = ceil((height * width * 4) * ((100 - percent) / 100))

while True:
    try:
      paint = int(input())
    except:
      print(f'{wall} quadratic m left.')
      break

    wall -= paint

    if wall <= 0:
      if wall >= 0:
        print(f'All walls are painted! Great job, Pesho!')
      else:
        print(f'All walls are painted and you have {abs(wall)} l paint left!')
      break






# '''
# 3
# 5
# 10
# 2
# 3
# 4
# Tired!
# '''

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 6 and 7 July 2019/4. Renovation/4. Renovation.js
function renovation(arg) {
  let [hight, weight, percent] = [...arg]
  let qm = (hight * weight * 4) * (100 - percent) / 100
  let num = 3
  while (arg[num]) {

    if (arg[num] == 'Tired!') {
      console.log(`${Math.ceil(qm)} quadratic m left.`)
      return
    }
    qm -= arg[num]
    num++
  }

  if (qm >= 0) {
    console.log(`All walls are painted! Great job, Pesho!`)
  } else {
    console.log(`All walls are painted and you have ${Math.abs(Math.ceil(qm))} l paint left!`)
  }
}

//renovation(['3', '5', '10', '2', '3', '4', 'Tired!']) 
